consumer/consumer.go

```
@@ -36,9 +38,18 @@ func (c *consumer) start(ctx context.Context) {
-			// TODO implement me
-			// Consumer should consume messages from the Kafka topic using the Kafka client
-			// and save the tick data to the data store
+			fetches := c.client.PollFetches(ctx)
+			if errs := fetches.Errors(); len(errs) > 0 {
+				// handle error
+			}
+			iter := fetches.RecordIter()
+			for !iter.Done() {
+				record := iter.Next()
+				var tick data.Tick
+				json.Unmarshal(record.Value, &tick)
```

golangci-lint (1.62.2)

Error return value of `json.Unmarshal` is not checked

(errcheck)